### PR TITLE
OWL-2635: Adding Stars to TopNav

### PR DESCRIPTION
**Pull request by chaospeach**
https://jira.uwo.ca/browse/OWL-2635

### CODE REVIEW

**Comment by fostersdesign** [comment]
Is there a sakai.property to enable/disable this feature?

File: reference/library/src/morpheus-owl/js/src/sakai.morpheus.more.sites.js [hunk 1]
```diff
@@ -392,6 +394,63 @@ $PBJQ(document).ready(function($){
...
+   * @param {*} event  - jQuery Event for item clicked
+   */
+  var topNavFavorite = function(event) {
+    var newFavId = $PBJQ(event.target).data("site-id")
+
+    getUserFavorites(function(list){
+        var favs = list; 
+        var ind = favs.indexOf(newFavId); 
+        
+        if(ind == -1) {
```
Comment (fostersdesign) [inline reference/library/src/morpheus-owl/js/src/sakai.morpheus.more.sites.js]: Wouldn't triple equals be better?

Comment (chaospeach) [reply reference/library/src/morpheus-owl/js/src/sakai.morpheus.more.sites.js]: That's an easy fix.

**Comment by chaospeach** [comment]
I'm not familiar with adding properties to Sakai, so there is currently no property to control this.

**Review by fostersdesign — APPROVED** [review]
Looks good, Adam. Thank you for making that change. We'll have to add a sakai.property for enabling it. I guess that can be in a separate PR. We'll get this first part in so it can be tested. Thanks.

**Pull request merged**